shell.nix

```diff
@@ -29,7 +29,7 @@ with pkgs;
       git
       opam
     ] else [ ]) ++
-    (with ocamlPackages; [ merlin ocamlformat utop ]);
+    (with ocamlPackages; [ ocaml-lsp merlin ocaml ocamlformat utop ]);
```

Comment:
is `ocaml` needed? It should come as a transitive dep

Reply:
Oh, you're fast. Fixed.